src/components/Kanban.tsx

```diff
@@ -274,11 +287,33 @@ const Kanban = ({
       );
       if (!predefinedSelection?.update) return;
       const { [`${columnKey}-uid`]: columnUid } = result;
+      const previousValue = toCellValue({
+        value: result[columnKey],
+        uid: columnUid?.toString(),
+      });
+      const isRemoveValue = column === `No ${columnKeySelection}`;
+      if (isRemoveValue && !previousValue) return;
       if (typeof columnUid !== "string") return;
+      if (!columnUid) {
+        createBlock({
+          parentUid: result.uid,
+          node: {
+            text: `${columnKeySelection}:: ${column}`,
+          },
+        }).then(onQuery);
+        return;
```

Comment:
move this into `predefinedSelection` with a flag that says `create: true` or something like that